src/main/java/dan200/computercraft/shared/wired/WiredNetwork.java

```diff
@@ -26,7 +26,7 @@
     HashSet<WiredNode> nodes;
     private HashMap<String, IPeripheral> peripherals = new HashMap<>();
 
-    public WiredNetwork( WiredNode node )
+    WiredNetwork( WiredNode node )
```

Comment:
On master. We need to review all our access modifiers at some point, but at least start here.